cmd/gcp-controller-manager/node_syncer.go

```diff
@@ -35,25 +36,27 @@ import (
 
 const (
 	nodeSyncerControlLoopName = "node-syncer"
-	nodeSyncerQueueName       = "node-syncer-queue"
 	nodeSyncerResyncPeriod    = 30 * time.Minute
+	nodeSyncerGSARemoveDelay  = 30 * time.Minute
 )
 
 // nodeSyncer implements a custom control loop responsible for synchronizing GCE with the list of
 // authorized and in-use GCP Service Accounts (GSA) on each Node.  GSA authorization is handled by
 // the service-account-verifier control loop which shares the result over the verifiedSAs map.  Use
 // of individual authorized GSA on each node is tracked by this control loop based on Pod events.
 type nodeSyncer struct {
```

Comment:
Existing nit but this struct and file should have more descriptive names.

Reply:
The name was originally chosen around HMS's SyncNode API.  Thinking of from the controller's perspective, perhaps nodeGSASyncer or gsaNodeSyncer is more appropriate?  Comments or suggestion?